spec/tracker/tracker_spec.rb

```diff
@@ -27,19 +27,19 @@ def app
         request = Rack::Request.new(env)
         case request.path
           when '/' then
-            [200, {'Content-Type' => 'application/html'}, ['<head>Hello world</head>']]
+            [200, {'Content-Type' => 'application/html'}, ['<head>Hello world</head>'.freeze]]
```

Comment:
Out of curiosity, do we also need all these freezers when we add `# frozen_string_literal: true` to the file? 🤔

Reply:
Good point...probably not. I'll test that out and update accordingly.